File: src/main/java/ua/com/lavi/usergenerator/utils/PasswordGenerator.java
package ua.com.lavi.usergenerator.utils;

import org.apache.commons.lang3.RandomStringUtils;

/**
 * Created by lavi on 15.03.2015.
 */
public class PasswordGenerator {

    private static int DEFAULT_PASSSWORD_LENGTH = 8;

    public static String generatePassword() {
        return RandomStringUtils.randomAlphanumeric(DEFAULT_PASSSWORD_LENGTH);
    }
}

File: src/main/kotlin/ua/com/lavi/usergenerator/utils/PasswordGenerator.kt
package ua.com.lavi.usergenerator.utils

import org.apache.commons.lang3.RandomStringUtils

/**
 * Created by Oleksandr Loushkin
 */

class PasswordGenerator {

    private val DEFAULT_PASSWORD_LENGTH = 8

    fun generatePassword(length: Int): String {
        return RandomStringUtils.randomAlphanumeric(length)
    }

    fun generatePassword(): String {
        return RandomStringUtils.randomAlphanumeric(DEFAULT_PASSWORD_LENGTH)
    }
}
